megamek/data/boards/buildings_no_basement/Schlucht 1.board

```diff
@@ -21,7 +21,7 @@ hex 0302 0 "" ""
 hex 0402 0 "" ""
 hex 0502 0 "" ""
 hex 0602 0 "" ""
-hex 0702 0 "building:1;bldg_cf:15;bldg_elev:2;bldg_basement_type:1;bldg_basement:1" ""
+hex 0702 0 "building:1;bldg_cf:15;bldg_elev:2;bldg_basement_type:1;bldg_basement_type:1" ""
```

Comment:
Umm, why do these have the field twice? The type was already there, are you sure these changes are correct?